### ISSUE TITLE
Grafana as data source for Grafana

### ISSUE THREAD

**Issue by knrdl** [grafana/grafana#52666]
The relevant measurements are written to a public grafana instance (readonly access).
Now I want them to be included into queries/panels within my private grafana instance.
Is this possible??? (Without copying the data)

Please help! Thanks a lot!

**Comment by torkelo** [comment]
no, there is currently no way to query a remote Grafana instance this way
